src/main/java/fi/otavanopisto/kuntaapi/server/integrations/ptv/AbstractPtvProvider.java

```diff
@@ -52,5 +84,565 @@
+    result.setLatitude(ptvServiceLocationChannel.getLatitude());
+    result.setLongitude(ptvServiceLocationChannel.getLongitude());
+    result.setCoordinateSystem(ptvServiceLocationChannel.getCoordinateSystem());
+    result.setCoordinatesSetManually(ptvServiceLocationChannel.getCoordinatesSetManually());
+    result.setPhoneServiceCharge(ptvServiceLocationChannel.getPhoneServiceCharge());
+    result.setWebPages(translateWebPages(ptvServiceLocationChannel.getWebPages()));
+    result.setServiceAreas(ptvServiceLocationChannel.getServiceAreas());
+    result.setPhoneChargeDescriptions(translateLanguageItems(ptvServiceLocationChannel.getPhoneChargeDescriptions()));
+    result.setAddresses(translateAddresses(ptvServiceLocationChannel.getAddresses()));
+    result.setChargeTypes(ptvServiceLocationChannel.getChargeTypes());
+    result.setServiceHours(translateServiceHours(ptvServiceLocationChannel.getServiceHours()));
+    result.setPublishingStatus(ptvServiceLocationChannel.getPublishingStatus());
+
+    return result;
+  }
+
+  protected WebPageChannel translateWebPageChannel(fi.otavanopisto.restfulptv.client.model.WebPageChannel ptvWebPageChannel) {
+    if (ptvWebPageChannel == null) {
+      return null;
+    }
+    
+    OrganizationId organizationKuntaApiId = translateOrganizationId(ptvWebPageChannel.getOrganizationId());
+    if (organizationKuntaApiId == null) {
+      return null;
+    }
+    
+    WebPageChannelId channelPtvId = new WebPageChannelId(PtvConsts.IDENTIFIFER_NAME, ptvWebPageChannel.getId());
+    WebPageChannelId channelKuntaApiId = idController.translateWebPageServiceChannelId(channelPtvId, KuntaApiConsts.IDENTIFIER_NAME);
+    if (channelKuntaApiId == null) {
+      logger.severe(String.format("Could not translate webPage channel id %s into Kunta API id", channelPtvId.getId()));
+      return null;
+    }
+    
+    WebPageChannel result = new WebPageChannel(); 
+    
+    result.setId(channelKuntaApiId.getId());
+    result.setType(ptvWebPageChannel.getType());
+    result.setOrganizationId(organizationKuntaApiId.getId());
+    result.setNames(translateLocalizedItems(ptvWebPageChannel.getNames()));
+    result.setDescriptions(translateLocalizedItems(ptvWebPageChannel.getDescriptions()));
+    result.setUrls(translateLanguageItems(ptvWebPageChannel.getUrls()));
+    result.setAttachments(translateAttachments(ptvWebPageChannel.getAttachments()));
+    result.setSupportContacts(translateSupportContacts(ptvWebPageChannel.getSupportContacts()));
+    result.setLanguages(ptvWebPageChannel.getLanguages());
+    result.setWebPages(translateWebPages(ptvWebPageChannel.getWebPages()));
+    result.setServiceHours(translateServiceHours(ptvWebPageChannel.getServiceHours()));
+    result.setPublishingStatus(ptvWebPageChannel.getPublishingStatus());
+    
+    return result;
+  }
+
+  private List<Address> translateAddresses(List<fi.otavanopisto.restfulptv.client.model.Address> ptvAddresses) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused private "translateAddresses" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)